Loop/Models/ServiceAuthentication/ShareService.swift

```diff
@@ -17,6 +17,8 @@ class ShareService: ServiceAuthentication {
     let title: String = NSLocalizedString("Dexcom Share", comment: "The title of the Dexcom Share service")
 
     init(username: String?, password: String?, url: URL?) {
+        // change the contents of the variable below to point to your custom share server
+        let customServer = ""
```

Comment:
Feel free to expand upon how to do this; like linking to your repo/blog.  Also, do you want to describe how to change the name as shown in the UI from "Custom"?